src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -662,17 +662,8 @@ export class Kernel implements IKernel {
             // Restart sessions and retries might make this hard to do correctly otherwise.
             notebook.session.registerCommTarget(Identifiers.DefaultCommTarget, noop);
 
-            // Request completions to warm up the completion engine (first call always takes a lot longer)
-            const completionPromise = this.requestEmptyCompletions();
```

Comment:
THe problem is we can see errors in the console, basically kernel is logging errors, however its an unhandled exception and the kernel is not able to send back a failure response.

Hence we end up waiting indefinitely for the promise to resolve as we don't ever get a response.
Fortunately in other places when we make this request, we have a timeout (to not wait and block VS Code for completions).